ci/jenkins/test.sh

```diff
@@ -541,7 +547,12 @@ function deliver_antrea {
     fi
     chmod -R g-w build/images/ovs
     chmod -R g-w build/images/base
-    DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
+    if [[ "$BUILD_TAG" != "latest" ]]; then
+        DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --build-tag ${BUILD_TAG} --pull
+        IMG_TAG="${BUILD_TAG}" ./hack/generate-manifest.sh
```

Comment:
I see other `./hack/generate-manifest.sh` invocations in this file. I think if multiple invocations of the script are possible for the same "code path", then some changes will be overwritten which is not correct. There should be a clear location where `./hack/generate-manifest.sh` is invoked, with all the right options, only once.